src/styles/components/input.scss

```diff
@@ -1,66 +1,68 @@
 @use '../abstracts/variables';
 @use '../abstracts/mixins';
 
-.flexInput {
+.miter-input {
   *,
   *:before,
   *:after {
     box-sizing: border-box;
   }
-  display: flex;
   align-items: flex-end;
-  padding-bottom: 7px;
   border-bottom: 1px solid variables.$secondary-hover;
-  padding-left: 8px;
-  padding-right: 8px;
+  display: flex;
   height: 53px;
-  &.active {
+  padding-bottom: 7px;
+
+  &--active {
     border-bottom: 1px solid variables.$primary;
   }
 
-  &.disabled {
+  &--disabled {
     border-bottom: 1px solid variables.$secondary-hover;
+
     *,
     *:before,
     *:after {
-      opacity: 60%;
       cursor: not-allowed;
+      opacity: 60%;
     }
   }
```

Comment:
This was already here but seems odd to me. Why add these styles to each element rather than the root element?

Reply:
Hm I'm not entirely clear on this myself but it may not be worth the time investigating tbh

Reply:
Ok i think the reason is that `cursor: not-allowed` doesn't propagate to the child elements for some reason, so it's only applied on the container (so the edge) but not the primary block. Ok to leave this as-is for now?

Reply:
yeah I was just curious